src/components/Breadcrumbs.js

```diff
@@ -7,26 +7,36 @@ const Breadcrumbs = ({ segments, separator }) => {
   return (
     <div
       css={css`
-        margin: 2em 0;
+        margin: 1rem 0 0;
+        padding: 1rem 2rem;
+
+        background-color: var(--secondary-background-color); ;
```

Comment:
Nit but to me this looks slightly too dark in Dark Mode 🤷 
<img width="1770" alt="Screenshot 2021-09-07 at 13 23 18" src="https://user-images.githubusercontent.com/17122543/132343864-282f9bbf-e9d5-4294-90c0-7b20106df004.png">